packages/flame/lib/src/particles/translated_particle.dart

```diff
@@ -1,27 +1,28 @@
 import 'dart:ui';
 
+import '../../extensions.dart';
 import '../components/mixins/single_child_particle.dart';
 import 'particle.dart';
 
-/// Statically offset given child [Particle] by given [Offset]
+/// Statically offset given child [Particle] by given [Vector2].
```

Comment:
this reads a bit off

Reply:
Suggestion for rewrite?

Reply:
no, I'm not even sure what it means haha. but it's existing stuff so feel free to ignore